### DISCUSSION TITLE
v2.0.0-alpha.1

### DISCUSSION THREAD

**Discussion by decahedron1** [pykeio/ort#120] [Announcements]
This is the first alpha release for `ort` v2.0. This update overhauls the API, improves performance, fixes bugs, and makes `ort` simpler to use than ever before.

Our shiny new website will help you migrate: https://ort.pyke.io/migrating/v2

Stuck? We're here to help:
- [Discord: `#💬｜ort-discussions`](https://discord.gg/uQtsNu2xMa)
- [GitHub Discussions](https://github.com/pykeio/ort/discussions)

A huge thank you to those who contributed to this release: [Ben Harris](https://github.com/bharrisau), [Ivan Krivosheev](https://github.com/ikrivosheev), [Rui He](https://github.com/ruihe774), [Andrea Corradi](https://github.com/acrrd), & [Lenny](https://github.com/JewishLewish)

Love `ort`? Consider [supporting us on Open Collective](https://opencollective.com/pyke-osai) 💖

❤️💚💙💛

<hr /><em>This discussion was created from the release <a href='https://github.com/pykeio/ort/releases/tag/v2.0.0-alpha.1'>v2.0.0-alpha.1</a>.</em>

**Comment by Liangdi** [comment]
Thank you for your great work. Does the onnxruntimer library I downloaded from pykeio support gpu? I used `cuda` feature, but it doesn’t seem to work

**Reply by decahedron1** [reply]
Yes, pyke provides CUDA binaries for both Windows & Linux in alpha 1.

You may need to run `cargo clean` if you used a version without CUDA previously. Are you also registering the EP in code? See https://ort.pyke.io/perf/execution-providers for more information on that.

If it still is not working, enable `tracing-subscriber` to debug whether the registration fails.
> Add `tracing-subscriber` to your Cargo.toml:
> ```toml
> [dependencies]
> tracing-subscriber = { version = "0.3", features = [ "env-filter", "fmt" ] }
> ```
>
> In your main function:
> ```rs
> fn main() {
>     tracing_subscriber::fmt::init();
> }
>```
>
>Set the environment variable `RUST_LOG` to `ort=debug` to see all debug messages from `ort`; this will look like:
> - Windows (PowerShell): `$env:RUST_LOG = 'ort=debug'; cargo run`
> - Windows (Command Prompt): use PowerShell ;)
> - macOS & Linux: `RUST_LOG="ort=debug" cargo run`

**Reply by Liangdi** [reply]
thanks
